torch/_refs/nn/functional/__init__.py

```diff
@@ -233,6 +236,26 @@ def selu(a: TensorLikeType, inplace: bool = False) -> TensorLikeType:
     return scale * torch.where(a > 0, a, rhs)
 
 
+# CompositeImplicitAutograd - don't register decomp
+def softmax(
+    a: TensorLikeType,
+    dim: Optional[int] = None,
+    dtype: Optional[torch.dtype] = None,
+) -> TensorLikeType:
```

Comment:
Missing `_stacklevel` argument.

```py
import torch
from torch._prims.context import TorchRefsMode
a = torch.randn(3, 3, device="cuda", dtype=torch.float32)
with TorchRefsMode(strict=True):
    torch.nn.functional.softmax(a, 1)
```
```py
TypeError: softmax() got an unexpected keyword argument '_stacklevel'
```

Reply:
I suggested removing it here:
https://github.com/pytorch/pytorch/pull/84502#discussion_r962989250
Do we need to keep these deprecation arguments in the refs? :/

Reply:
Yes, as you see there's no way to use `torch.nn.functional.softmax` under TorchRefsMode without this argument.

Reply:
added a test for this behavior in "Test aliases when testing refs". if it's too much for this PR, i can just revert it later, once we see that ci passes. as of now, aliases are not tested within the context afaict

Reply:
upd: mario said it's okay to keep these alias tests. the only thing i'm concerned about is that there'll be a bit of duplication during testing. eg if you have separate refinfos for functional softmax, special softmax, and softmax, all those end up tested against the same softmax opinfo, so all aliases are tested for each of those as well. but perhaps it's fine(?) because doing some fancy filtering of those might be error-prone

Reply:
How could you have separate refinfos for softmax and special softmax? We shouldn't allow to register refinfos for aliases if we can avoid it.

If you think we could write a test for that, that could go in another PR.